src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -165,6 +165,9 @@ class SnowflakeDialect(default.DefaultDialect):
     # and denormalize_name() must be provided.
     requires_name_normalize = True
 
+    # READ_COMMITTED is the only supported isolation level
+    default_isolation_level = "READ_COMMITTED"
```

Comment:
this is one sample fix (or should not be a fix) to set the default isolation level of our dialect